pkg/api/testapi/testapi.go

```diff
@@ -258,6 +260,15 @@ func init() {
 			externalTypes:        api.Scheme.KnownTypes(externalGroupVersion),
 		}
 	}
+	if _, ok := Groups[authorization.GroupName]; !ok {
```

Comment:
It'd be nice to have a unit test somewhere that makes sure this parallel registration stays in sync.